pyfakefs/fake_pathlib.py

```diff
@@ -496,10 +504,10 @@ def open(self, mode='r', buffering=-1, encoding=None,
         """
         if self._closed:
             self._raise_closed()
-        return FakeFileOpen(self.filesystem)(
+        return FakeFileOpen(self.filesystem, use_io=True)(
```

Comment:
I added `use_io=True` 3 times in this file to make some py27 tests pass on Windows.  Not sure if it's the best approach, maybe I should add something to the `self._use_io` logic in `FakeFileOpen.__init__()`  instead?

The test failures can be seen in [AppVeyor](https://ci.appveyor.com/project/jmcgeheeiv/pyfakefs/build/3.4.172/job/3pbw1m03nw1swqg2), e.g:
```
Traceback (most recent call last):
  File "pyfakefs\tests\fake_filesystem_unittest_test.py", line 131, in test_fakepathlib
  File "pyfakefs\fake_pathlib.py", line 508, in open
  File "pyfakefs\fake_filesystem.py", line 4765, in __call__
TypeError: _call_ver2() takes at most 6 arguments (7 given)
```
```
Traceback (most recent call last):
  File "pyfakefs\tests\fake_pathlib_test.py", line 508, in test_read_text
    self.assertEqual(file_path.read_text(), 'foo')
  File "pyfakefs\fake_pathlib.py", line 527, in read_text
    errors=errors) as f:
  File "pyfakefs\fake_filesystem.py", line 4765, in __call__
    return self._call_ver2(*args, **kwargs)
TypeError: _call_ver2() got an unexpected keyword argument 'errors'
```